frontend/package.json

```diff
@@ -32,6 +32,7 @@
     "dotenv": "^8.2.0",
     "glob-all": "^3.2.1",
     "jest-junit": "^7.0.0",
+    "jest-watch-typeahead": "^2.2.0",
```

Comment:
oh we can remove this one, it's a Dev dependency already.

Reply:
Nice catch, I forgot about that guy. It is now gone in the latest commit.